ClosedXML_Tests/Excel/CalcEngine/ExpressionTests.cs

```diff
@@ -0,0 +1,44 @@
+﻿using ClosedXML.Excel;
+using ClosedXML.Excel.CalcEngine;
+using ClosedXML.Excel.CalcEngine.Exceptions;
+using NUnit.Framework;
+using System;
+using System.Globalization;
+using System.Threading;
+
+namespace ClosedXML_Tests.Excel.CalcEngine
+{
+    [TestFixture]
+    public class ExpressionTests
+    {
+        [OneTimeSetUp]
+        public void SetCultureInfo()
+        {
+            Thread.CurrentThread.CurrentCulture = CultureInfo.CreateSpecificCulture("en-US");
+        }
+
+        [Test]
+        public void UnaryExpression()
```

Comment:
In this test the expected and actual parameters are the wrong way around. First parameter should be expected value.